spec/controllers/annotations_controller_spec.rb

```diff
@@ -1,11 +1,16 @@
 require 'rails_helper'
 require "cancan/matchers"
 require "list_annotations_map"
+require './spec/support/anno_auth_helper'
 
 include Warden::Test::Helpers
 Warden.test_mode!
 include Devise::TestHelpers
 
+RSpec.configure do |c|
```

Comment:
See my other comment for how I think you can include this module slightly differently.